docs/examples/fetch_models.sh

```diff
@@ -40,3 +40,10 @@ wget -O /tmp/inception_v3_2016_08_28_frozen.pb.tar.gz \
      https://storage.googleapis.com/download.tensorflow.org/models/inception_v3_2016_08_28_frozen.pb.tar.gz
 (cd /tmp && tar xzf inception_v3_2016_08_28_frozen.pb.tar.gz)
 mv /tmp/inception_v3_2016_08_28_frozen.pb model_repository/inception_graphdef/1/model.graphdef
+
+# Custom models
+mkdir -p model_repository/image_preprocess_nchw_3x224x224_inception/1
+mkdir -p model_repository/preprocess_resnet50_ensemble/1
```

Comment:
Move this line to "# Ensemble models" perhaps with a note saying "Create empty version directories for ensemble models"